components/brave_wallet/common/switches.h

```diff
@@ -14,6 +14,7 @@ inline constexpr char kDevWalletPassword[] = "dev-wallet-password";
 
 // Ratios service dev URL
 inline constexpr char kAssetRatioDevUrl[] = "asset-ratio-dev-url";
+inline constexpr char kMeldAssetRatioDevUrl[] = "meld-asset-ratio-dev-url";
```

Comment:
What's the purpose of this switch?

Reply:
Meld API has sandbox mode, in that case it should use different Url.